src/Module/Template/Step/ValidateTemplateDirectory.php

```diff
@@ -14,7 +14,7 @@
  */
 class ValidateTemplateDirectory implements Step
 {
-    const DEFAULT_TEMPLATE_DIRECTORY = 'website';
+    public const DEFAULT_TEMPLATE_DIRECTORY = 'website';
```

Comment:
Why make this const public when the others are private ?

Reply:
It's because this const is also used here: https://github.com/CouscousPHP/Couscous/blob/c168d0b9c1449d60f2a31b2a30c7ab7e61e419be/src/Module/Template/Step/UseDefaultTemplate.php#L53